src/components/ApplicationForm/Skills.js

```diff
@@ -275,6 +268,27 @@ export default ({ refs, errors, formInputs, onChange, role, handleResume }) => {
             customRef={refs['longAnswers2Ref']}
           />
         </FormGroup>
+        <FormGroup>
+          <QuestionHeading>question 21</QuestionHeading>
+          <SubHeading size="1.25em">
+            What character (from a movie, show, book, etc.) do you relate to most and why? (max 50
+            words)
+            <Required />
+          </SubHeading>
+          <StyledTextArea
+            maxWords="200"
```

Comment:
I think this needs to be updated to 50 words too

Reply:
lol u right